src/components/sections/preface/section_preface.less

```diff
@@ -73,11 +73,8 @@
     text-transform: uppercase;
   }
 
-  .card {
-    margin-bottom: 4.2rem;
-  }
-
   .scroll-down {
+    margin-top: 4.2rem;
```

Comment:
It's better to use margin-bottom when it's possible. Try to replace with "media-preview { margin-bottom: 4.2rem; }"